USAGE.md

```diff
@@ -179,7 +182,7 @@ The following YAML shows the supported key–value pairs for an author:
 ```yaml
 github: dhimmel  # strongly suggested
 name: Daniel S. Himmelstein  # mandatory
-initials: DSH  # strongly suggested
+initials: DSH  # optional
```

Comment:
initials are not used anywhere by default, so seems excessive to call them "strongly suggested"

Reply:
Agreed.  I think that was a remnant from deep review where we were trying to automate things like funding statements.  We haven't needed them in more modern forms of Manubot.